adbmanager/adbmanager.go

```diff
@@ -46,3 +46,31 @@ func (model Model) UnlockDevice(serial string) error {
 	keyEvent1Cmd := model.cmdFactory.Create(model.binPth, []string{"-s", serial, "shell", "input", "1", "&"}, nil)
 	return keyEvent1Cmd.Run()
 }
+
+// InstallAPKCmd builds and returns a `Command` for installing APKs on an attached device or emulator.
+// The `Command` can than be run by the consumer without needing to know the implementation details.
+func (model Model) InstallAPKCmd(pathToAPK string, commandOptions *command.Opts) command.Command {
+	cmd := model.cmdFactory.Create(model.binPth, []string{"install", pathToAPK}, commandOptions)
+	return cmd
+}
+
+// RunInstrumentedTestsCmd builds and returns a `Command` for running instrumented tests on an attached device or emulator.
+// The `Command` can than be run by the consumer without needing to know the implementation details.
+func (model Model) RunInstrumentedTestsCmd(
```

Comment:
NOTE: Breaking the functions parameters with newlines is a bit unusual to our codebase.
I only know about a few places where we do this, those cases we have more parameters (and would be better to use a parameter struct).

Probably i would write these params in a single line.

Reply:
NOTE: i was thinking of the `additionalTestingOptions` argument.
I guess it is not always needed, but because of the function signature, on the consumer side we always need to provide it (sometimes just a nil).

Similar parameters can be specified as a variadic argument as the last argument, like: `func do(args ...string)`. You can call this with and without args:
- `do()`
- `do(a, b, c)`

This case is a bit more complicated as `additionalTestingOptions` belongs to `packageName` and `testRunnerClass` args, would be great to keep them close, but at the same time it is logical to start with the main params and leave `command.Opts` at the end.

Reply:
> Breaking the functions parameters with newlines is a bit unusual to our codebase.

This is another carryover from Swift, this format is pretty common there. I like it a little better, but I'm ok moving to a single line. Is there a linter check for this to ensure it's consistent?

> This case is a bit more complicated as additionalTestingOptions belongs to packageName and testRunnerClass args, would be great to keep them close, but at the same time it is logical to start with the main params and leave command.Opts at the end.

You mentioned above about a parameter struct, what would you think about that as a solution? I could make an `InstrumentedTestParams` struct that has a constructor with the variadic testing options like you mentioned, and then have that being passed into the `RunInstrumentedTestsCmd()` method along with the `command.Opts`.

Reply:
Alrighty, I'll leave it for now and we can refactor later if needed 👍